### PR TITLE
Build local-images for aws iam and make shasums idempotent

### PR DESCRIPTION
**Pull request by TerryHowe**
Run checksums at end so we get all of them.

### CODE REVIEW

File: projects/kubernetes/kubernetes/build/create_release_checksums.sh [hunk 1]
```diff
@@ -42,7 +42,7 @@ rm -f $SHA256SUM
 rm -f $SHA512SUM
 echo "Writing artifact hashes to SHA256SUM/SHA512SUM files..."
 cd $ASSET_ROOT
-for file in $(find ${ASSET_ROOT} -type f ); do
+for file in $(find ${ASSET_ROOT} -type f | grep -v '\.sha[25][51][62]' ); do
```
Comment (abhay-krishna) [inline projects/kubernetes/kubernetes/build/create_release_checksums.sh:45]:
```suggestion
for file in $(find ${ASSET_ROOT} -type f -not -path '*\.sha[25][51][62]' ); do
```
nit: Just avoiding pipes and another exit code to track. Thinking if we could use this
```bash
(~/test_find) touch test.txt
(~/test_find) touch test.py
(~/test_find) touch test.sha256
(~/test_find) touch test.sha512                                                                                                                                                                                                    
(~/test_find) find ./ -type f -not -path '*\.sha[25][51][62]'
./test.py
./test.txt
```

**Review by bnrjee — APPROVED** [review]

**Pull request merged**